fog/view/server/tests/streaming_smoke_tests.rs

```diff
@@ -215,22 +215,74 @@ fn test_streaming_integration(omap_capacity: u64, store_count: usize, blocks_per
     assert_eq!(result.highest_processed_block_count, 6);
     assert_eq!(result.next_start_from_user_event_id, 4);
     assert_eq!(result.rng_records.len(), 0);
-    assert_eq!(result.tx_out_search_results.len(), 3);
+    assert_eq!(result.fixed_tx_out_search_results.len(), 3);
     {
-        let sort_txs = mc_fog_view_server_test_utils::interpret_tx_out_search_results(
-            result.tx_out_search_results.clone(),
-        );
+        let mut sort_txs = result.tx_out_search_results.clone();
+        sort_txs.sort_by(|x, y| x.search_key.cmp(&y.search_key));
         assert_eq!(sort_txs[0].search_key, vec![1u8; 16]);
         assert_eq!(sort_txs[0].result_code, 1);
         assert_eq!(sort_txs[0].ciphertext, vec![1u8; 232]);
+        assert_eq!(sort_txs[0].padding, vec![0u8; 23]);
 
         assert_eq!(sort_txs[1].search_key, vec![2u8; 16]);
         assert_eq!(sort_txs[1].result_code, 1);
         assert_eq!(sort_txs[1].ciphertext, vec![2u8; 232]);
+        assert_eq!(sort_txs[1].padding, vec![0u8; 23]);
 
         assert_eq!(sort_txs[2].search_key, vec![3u8; 16]);
         assert_eq!(sort_txs[2].result_code, 1);
         assert_eq!(sort_txs[2].ciphertext, vec![3u8; 232]);
+        assert_eq!(sort_txs[2].padding, vec![0u8; 23]);
+    }
+    {
+        let mut sort_fixed_txs = result.fixed_tx_out_search_results.clone();
+        sort_fixed_txs.sort_by(|x, y| x.search_key.cmp(&y.search_key));
+        let expected_payload_length = 232;
+
+        assert_eq!(sort_fixed_txs[0].search_key, vec![1u8; 16]);
+        assert_eq!(sort_fixed_txs[0].result_code, 1);
+        assert_eq!(
+            sort_fixed_txs[0].ciphertext[0..expected_payload_length],
```

Comment:
:wrench: It may have been done to be more explicit, but consider removing the leading 0, and in others
```suggestion
            sort_fixed_txs[0].ciphertext[..expected_payload_length],
```

Reply:
Done.